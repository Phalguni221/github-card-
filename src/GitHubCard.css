body {
  background-image: url('Light-Pink-Background.jpg');
}
h1{
  color:blue;
  text-align:center;
}

h2 {
  color:black;
  text-align:center;
}

