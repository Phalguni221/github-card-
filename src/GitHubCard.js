import './GitHubCard.css';
import PersonalHeadshot2 from './PersonalHeadshot2.jpg'


function GitHubCard() {
    return (
      <div>
        <body>
<h1>Github Username: Phalguni221</h1>
          <img src={PersonalHeadshot2}/>
          <br></br>
          <br></br>
          <h2>
          I am currently an ambitious and hardworking student at the NJIT Full Stack Development Boot Camp.
          I hope to be a future web developer and put to use the skills that I have learned over the course of the boot camp. 
          More specifically, I want to be able to incorporate creativity with practical technical skills in the field of computer programming.
          </h2>
        </body>
              </div>
    );
  }

export default GitHubCard;
