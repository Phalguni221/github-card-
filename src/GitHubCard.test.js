import { render, screen } from '@testing-library/react';
import GitHubCard from './GitHubCard';
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const plant = renderer.create(<GitHubCard/>).toJSON()
  expect(plant).toMatchSnapshot()
})