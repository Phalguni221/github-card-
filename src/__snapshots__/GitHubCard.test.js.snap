// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div>
  <body>
    <h1>
      Github Username: Phalguni221
    </h1>
    <img
      src="PersonalHeadshot2.jpg"
    />
    <br />
    <br />
    <h2>
      I am currently an ambitious and hardworking student at the NJIT Full Stack Development Boot Camp. I hope to be a future web developer and put to use the skills that I have learned over the course of the boot camp. 
      More specifically, I want to be able to incorporate creativity with practical technical skills in the field of computer programming.
    </h2>
  </body>
</div>
`;
